duckduckgo/Android
# Feature/autocomplete

Asana Issue URL: https://app.asana.com/0/488551667048375/492750258120867

## Description
Implement autocomplete suggestions for URLs and queries entered into the omnibar.


## Steps to Test this PR:
1. Start typing a query
1. Ensure autocomplete suggestions are populated, and update as you type
1. Delete the whole query; ensure the suggestions disappear
1. Type in a large string of gibberish which will result in no results; ensure "no suggestions" is shown
1. Type in "cnn.com" as the search query; ensure you see a globe in the results, indicating it's a URL rather than a query
1. For any results, ensure tapping the `+` button moves the suggestion to the omnibar and allows it be edited
